src/ORM/OrmCache.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         3.5.0
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\ORM;
+
+use Cake\Cache\Cache;
+use Cake\Datasource\ConnectionManager;
+
+/**
+ * ORM Cache.
+ *
+ * This tool is intended to be used by deployment scripts so that you
+ * can prevent thundering herd effects on the metadata cache when new
+ * versions of your application are deployed, or when migrations
+ * requiring updated metadata are required.
+ */
+class OrmCache
+{
+
+    /**
+     * Schema
+     *
+     * @var \Cake\Datasource\SchemaInterface
```

Comment:
Shouldn't the type be same as those returned by `getSchema()` as it's return value is assigned to this property?